Sources/MapboxMaps/Style/Style.swift

```diff
@@ -721,6 +725,71 @@ public final class Style: StyleProtocol {
         }
     }
 
+    /// Adds an image to be used in the style.
+    ///
+    /// This API can also be used for
+    /// updating an image. If the image id was already added, it gets replaced
+    /// by the new image.
+    ///
+    /// The image can be used in
+    /// [`icon-image`](https://www.mapbox.com/mapbox-gl-js/style-spec/#layout-symbol-icon-image),
+    /// [`fill-pattern`](https://www.mapbox.com/mapbox-gl-js/style-spec/#paint-fill-fill-pattern), and
+    /// [`line-pattern`](https://www.mapbox.com/mapbox-gl-js/style-spec/#paint-line-line-pattern).
+    ///
+    /// - Parameters:
+    ///   - image: Image to add.
+    ///   - id: ID of the image.
+    ///   - sdf: Option to treat whether image is SDF(signed distance field) or not.
+    ///         Setting this to `true` allows template images to be recolored. The
+    ///         default value is `false`.
+    ///   - content: An array of four numbers, with the first two specifying the
+    ///         left, top corner, and the last two specifying the right, bottom
```

Comment:
> content: An array of four numbers

Seems ambiguous. I see an `ImageContent` type and not the array of numbers.

Reply:
Just saw our comment:
> I decided to leave the image content as a parameter accessible, if needed, to the user. To make the API more Swifty the parameter type could be `UIEdgeInsets` instead of `ImageContent`, as edge insets should be more familiar to iOS developers.

Switching to `UIEdgeInsets` looks legit for me. It makes much more sense for non-SDK developers. 
However, we have to be sure that it wouldn't prevent developers from passing this value to some other method or variables that accepts only `ImageContent` type.

Reply:
I support using UIEdgeInsets. We could name the parameter contentInsets as well, which has precedent in UIKit APIs.